e2e/ctl_v3_watch_test.go

```diff
@@ -116,3 +116,30 @@ func ctlV3Watch(cx ctlCtx, args []string, kvs ...kv) error {
 	}
 	return proc.Stop()
 }
+
```

Comment:
share setup code from `cmdArgs :=` to `if cx.interactive { ... }` instead of duplicating: `ctlV3WatchProc(cx ctlCtx, args []string) (*expect.ExpectProcess, error)`